packages/contracts/utils/structEncoding.ts

```diff
@@ -0,0 +1,24 @@
+import * as ethers from "ethers";
+
+export function structDefinitionToEncoding(structDefinition: string): string {
```

Comment:
The name of this is a little confusing when coupled with  `encodeStruct`, since this function doesn't do the ABI encoding.

Maybe we should change the name to `canonicalizeStruct` or something else.

Reply:
Yup, don't like the name either. `abiEncodingForStruct`?